### ISSUE TITLE
as_resizableRoundedImage border doesn't respect roundedCorner argument

### ISSUE THREAD

**Issue by ArEnSc** [facebook/AsyncDisplayKit#2903]
ASDK 2.0.1

When choosing which corners to round with the border option, the border is not taken into account.
i.e

```swift

 UIImage.as_resizableRoundedImage(withCornerRadius: 10.0, cornerColor: UIColor.red, fill: UIColor.yellow, borderColor: UIColor.gray, borderWidth: 0.5, roundedCorners: [.topLeft,.topRight], scale: 0.0)

```
results in the border to still have rounded corners with top right and top left of the corners of the rectangle not clipped.

expected that the corner rounding of the solid object.

**Comment by hannahmbanana** [comment]
@ArEnSc - thanks for pointing out this bug! I'll put up a fix for this shortly.

**Comment by hannahmbanana** [comment]
Resolved in #2905.
